com.moonlight_stream.Moonlight.json

```diff
@@ -0,0 +1,50 @@
+{
+	"app-id": "com.moonlight_stream.Moonlight",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "5.11",
+	"sdk": "org.kde.Sdk",
+	"command": "moonlight",
+	"rename-icon": "moonlight",
```

Comment:
not a blocker, but if flatpak is your primary then it's normally better to have your icons with exportable names rather than rename them (saves accidentally missing assets if you hard code it somewhere)

Reply:
yeah, I did that with the other resources. The icon was more problematic because that same SVG is actually bundled into the app via QRC file for all platforms and used to set the window icon at runtime.